demo/unconditional_demo.py

```diff
@@ -65,7 +65,7 @@ def main():
     if args.sample_cfg is None:
         args.sample_cfg = dict()
 
-    results = sample_uncoditional_model(model, args.num_samples,
+    results = sample_unconditional_model(model, args.num_samples,
```

Comment:
You may also fix the api name here. https://github.com/open-mmlab/mmgeneration/blob/e1212d53d8bc1a39ba8003070e1f93d835d95312/demo/unconditional_demo.py#L12